gilbreth_perception/config/model_list.yaml

```diff
@@ -0,0 +1,15 @@
+part_list:
+ - name: disk 
+   path: /model/disk.pcd
+   pick_point_index: 137
```

Comment:
what's pick point index?

Reply:
It is the index of the pick point in the point cloud data, for example, 137 means the pick point is the 137th point in the disk.pcd file.
But it is not important now, as I changed it to x, y, z.